multi-int-store/src/main/java/com/connexta/multiintstore/config/S3StorageConfiguration.java

```diff
@@ -4,7 +4,7 @@
  * Released under the GNU Lesser General Public License version 3; see
  * https://www.gnu.org/licenses/lgpl-3.0.html
  */
-package com.connexta.ingest.config;
+package com.connexta.multiintstore.config;
```

Comment:
Do `docker-compose.yml` and the readme need to be updated for the new config values?

Reply:
Thanks, I updated them in the latest commit.